### DISCUSSION TITLE
Is there an path preserving policy that will refuse to move files to a different drive?

### DISCUSSION THREAD

**Discussion by dagelf** [trapexit/mergerfs#1195] [General]
I'm quite frankly a bit overwhelmed and didn't have enough sleep to figure this out from reading all the policies. I will answer my own question when I look at this with a fresh brain.

**Comment by trapexit** [comment]
I don't know what you mean. No policy moves files. Policies *pick which branch to use. That's it. The only movement mergerfs does is with regard to the `moveonenospc` feature.
